src/composable/Questionnaire.ts

```diff
@@ -55,7 +55,10 @@ export class Questionnaire implements IQuestionnaire {
     Object.assign(this, data);
 
     // Create a new collection for our flow logic
-    this.steps = [...this.questions];
+    this.steps = this.questions.map((q, i) => ({
```

Comment:
Maybe a longer comment? It seems like this just adds an `order` to the elements to preserve the order in the array?

Reply:
Ahh, I see the comment below where `order` is declared! Maybe something similar here?

Reply:
Technically, this is no longer required--just a remnant from one of many attempts to fix this. Doesn't hurt anything now; will circle back and gut it later.